meinberlin/assets/scss/components/_accordion.scss

```diff
@@ -38,7 +38,7 @@
 }
 
 .accordion__body {
-    padding: $padding / 2;
+    padding: $padding/2 1em;
```

Comment:
Yeah! That looks much nicer! You could use $padding here instead of 1em, as that is also used in the first bit.